src/main/java/pl/put/poznan/buildingInfo/logic/locations/Room.java

```diff
@@ -1,14 +1,26 @@
 package pl.put.poznan.buildingInfo.logic.locations;
 
 import lombok.*;
+import lombok.experimental.SuperBuilder;
 import pl.put.poznan.buildingInfo.logic.visitors.Visitor;
 
 @Getter
 @Setter
+@AllArgsConstructor
+@NoArgsConstructor
+@SuperBuilder
 public class Room extends AbstractLocation {
+    protected Float area;
+    protected Float cube;
+    protected Float lightPower;
```

Comment:
dlaczego nie private?

Reply:
Bo kopiowałem z tej starej wersji klasy AbstractLocation. Wydaje mi się, że w niczym to nie szkodzi, ale możemy zmienić.

Reply:
Też uważam, że nie szkodzi. Zastanawiałem się czy może stoi za tym jakaś głębsza idea.